src/python/pants/option/global_options.py

```diff
@@ -733,6 +734,18 @@ def register_bootstrap_options(cls, register):
                 "Add a '*' to the list to capture all known scopes."
             ),
         )
+        register(
+            "--stats-complete-async",
```

Comment:
not sure why the stats recording name is here....
this is for streaming work unit handler plugins, no?

Reply:
True. We already have `--streaming-workunits-report-interval"`. Maybe `--streaming-workunits-complete-async`?